### PR TITLE
Implement introActivity with webview (#62)

### PR DESCRIPTION
**Pull request by tommy5oo**

### CODE REVIEW

File: app/src/main/java/com/skt/nugu/sampleapp/activity/LoadingActivity.kt [hunk 1]
```diff
@@ -62,10 +62,20 @@ class LoadingActivity : AppCompatActivity(), ClientManager.Observer {
...
+
+    override fun onResume() {
+        super.onResume()
         /** Add observer When initialized, [onInitialized] is called **/
         ClientManager.observer = this
     }
 
+    override fun onDestroy() {
+        /** remove observer **/
+        ClientManager.observer = null
```
Comment (shshshl09) [inline app/src/main/java/com/skt/nugu/sampleapp/activity/LoadingActivity.kt]: ClientManager.observer = null  at onPaused() ?

File: app/src/main/java/com/skt/nugu/sampleapp/activity/IntroActivity.kt [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            domStorageEnabled = true
+            javaScriptEnabled = true
+        }
+        webView.webViewClient = object : WebViewClient() {
+            override fun shouldOverrideUrlLoading(view: WebView?, url: String?): Boolean {
+                url?.apply {
+                    // Otherwise, the link is not for a page on my site, so launch
+                    // another Activity that handles URLs
+                    val intent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
+                    startActivity(intent)
```
Comment (shshshl09) [inline app/src/main/java/com/skt/nugu/sampleapp/activity/IntroActivity.kt]: handle ActivityNotFoundException

Comment (tommy5oo) [reply app/src/main/java/com/skt/nugu/sampleapp/activity/IntroActivity.kt]: url is a known scheme, so no problem
But I will add clearly

Comment (tommy5oo) [reply app/src/main/java/com/skt/nugu/sampleapp/activity/IntroActivity.kt]: done

File: app/src/main/java/com/skt/nugu/sampleapp/activity/LoadingActivity.kt (hunk 1 above)
Comment (tommy5oo) [reply app/src/main/java/com/skt/nugu/sampleapp/activity/LoadingActivity.kt]: done

**Pull request merged**